src/main/java/br/unb/cic/analysis/df/OverridingAssignmentAnalysis.java

```diff
@@ -51,21 +49,13 @@ public OverridingAssignmentAnalysis(Body methodBody, AbstractMergeConflictDefini
 
                     for (Local useBox: getUseVariables(u)){
                         String useBoxName = getLocalName(useBox);
-                        if (defInName.equals(useBoxName) && defInName.equals(localName)){
+                        if (!defInName.equals(useBoxName) && defInName.equals(localName)){
```

Comment:
Because this "!" was added here?

Reply:
The case x = x + ...; does not generate conflict. So the right side doesn't have to be the equals.

Reply:
I updated this

Reply:
There is no need to analyze the UseBox of a definition (DefBox). Example: x = ...; no matter what comes on the right of the statement.